### ISSUE TITLE
Taskbar shortcuts for existing PWAs broken after update

### ISSUE THREAD

**Issue by shortstopryan** [filips123/PWAsForFirefox#425]
Taskbar shortcuts broken after new update with existing PWAs.
Previously, I could create a PWA, open that PWA, pin that open window to the task bar, then access it via clicking the shortcut on the taskbar. After update, all current PWA shortcuts seem to be broken. Says it cannot locate "firefoxpwa.exe".

If I delete shortcut from taskbar, then re-pin it, still broken with same error message.

If I delete PWA, then create new PWA from same site, then add to task bar, it seems to work.

I have 15 PWAs installed, would like to not have to re-create them all.

Windows 10
64-bit architecture
PWA extension info:
    Extension version: 2.9.0
    Native version: 2.9.0
    Runtime version: 119.0.1
    Firefox version: 120.0

**Comment by shortstopryan** [comment]
Please delete, I was doing something wrong

**Comment by filips123** [comment]
How did you install the previous version and how the latest (using MSI installer, winget, chocolately, etc.)? Also, did you install it per-user or per-machine?

**Comment by shortstopryan** [comment]
Gotcha, thats a point I hadnt considered, I may have done it per-machine previously. I def did it per-user this time. Both times using the MSI installer.

Firefoxpwa works, and its pointing to the local appdata folder for my user account.

**Comment by shortstopryan** [comment]
Let me know if there are indeed manual steps to fix this. I need the shortcuts working asap (I use this for work) so I'm at the point where I may just rebuild them.

**Comment by shortstopryan** [comment]
Perfect. I uninstalled via appwiz.cpl, then ran the MSI again and chose Per-Machine install this time. This makes it so any PWAs added Pre-Update are able to be pinned to taskbar and opened via clicking the taskbar shortcut. 

I did have to re-setup a couple of the PWAs I setup Post-Update and pin them using the new install, but that was to be expected, since those were setup as Per-User.

Thanks for your assistance.
